assets/templates/carousel-component.html

```diff
@@ -78,14 +79,77 @@ <h3>Carousel Component</h3>
             <hr class="dpsg-break" />
             <div class="dp-container">
               <div class="dp-rowflex">
-                <div class="col-sm-6 m-b-12">
-                  <div class="dp-box-shadow dpsg-content-sample">
-                    <p class="dp-boxshadow">Carousel 1</p>
+                <div class="col-sm-12 col-md-6">
+                  <div class="dp-carousel dp-carousel-orange">
+                    <div class="dp-carousel-wrapper">
```

Comment:
Why not add the color in the wrapper? Instead of targeting the wrapper from above.
```suggestion
                    <div class="dp-carousel-wrapper dp-carousel-orange">
```

Reply:
I thought of it like this in case an element is added outside the wrapper that needs the color format. So I think it is better to add it in the parent element.

Reply:
This is a case of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), when we need this we will think about it. 😄

Reply:
Ok I change it!